### ISSUE TITLE
Bug (CLI

### ISSUE THREAD

**Issue by Roadrunner2058** [att-comdev/armada#173]
**Is this a bug report or feature request?** (choose one):

**Python Version** (output of `python --version`):

**Development or Deployment Environment?**:

**Release Tag or Master**:

**Expected Behavior**:

**What Actually Happened**:

**How to Reproduce the Issue** (as minimally as possible):

**Any Additional Comments**:

**Comment by gardlt** [comment]
I have to agree on the UX for these files should be updated to follow a nice format other than --[option]. 

I would like to make the cli group grow in the command options to improve it. 

Example:

```
armada test --file=examples/simple.yaml  to armada test file examples/simple.yaml
```
